packages/shared/scope-hoisting/src/link.js

```diff
@@ -374,10 +376,14 @@ export function link({
               mod,
               'js',
             );
-            if (!isUnusedValue(path) || isReferenced) {
-              let name = assertString(mod.meta.exportsIdentifier);
+            let used = !isUnusedValue(path);
+
```

Comment:
nit: maybe a newline above `let used` rather than in between these?

Reply:
`isReferenced` and `isValueUsed` are closely related, so I wanted to keep them together, but I moved them closer to the if condition.